interface/i18n/mist.nl.i18n.json

```diff
@@ -235,40 +235,40 @@
                     "wrongPassword": "Incorrect wachtwoord.",
                     "importFailed": "Kon bestand niet inporteren en kreeg: __error__"
                 },
-                "viaShapeshift": "Instant conversion via <a href='https://shapeshift.io/' target='_blank'>Shapeshift</a>",
-                "syncMessage": "Block __displayBlock__ of __highestBlockString__",
-                "syncMessageWithStates": "Block __displayBlock__ of __highestBlockString__ (Chain structure __statesPercent__%)"
+                "viaShapeshift": "Instant conversie via <a href='https://shapeshift.io/' target='_blank'>Shapeshift</a>",
```

Comment:
A bit too literal. I'd go for:

> Direct omwisselen via Shapeshift

Since a currency exchange is a 'wisselkantoor'. 'Conversie' is rarely use in this context.